recipes/mdsapt/meta.yaml

```
@@ -0,0 +1,58 @@
+{% set name = "mdsapt" %}
+{% set version = "2.0.3_rc1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/calpolyccg/MDSAPT/archive/v2.0.3-rc1.tar.gz
+  sha256: 4d90626d9494a8531d9190c5d5b41a9e33fcbc11f6138336494b402fa59dd450
+
+build:
+  skip: True  # [win or osx or py < 39]
```

This can probably be noarch.
```suggestion
  noarch: python
```

---

No, it can't, because it's not supported on windows, and [the bot complained that it can't be noarch because of that](https://github.com/conda-forge/staged-recipes/pull/24782#issuecomment-1953272625). However, I should change it to support osx, because it does.